lib/schemaGetters.js

```diff
@@ -103,15 +100,15 @@ const objectIdType = () => {
     type: 'string',
     description: 'Hexadecimal identifier of the document in the collection',
     pattern: '^[a-fA-F\\d]{24}$',
-    example: new ObjectId().toString(),
+    example: '000000000000000000000000',
   }
 }
 
 const stringType = () => ({
   type: 'string',
   pattern: '^(?!\\s*$).+',
   description: 'String identifier of the document in the collection',
-  example: uuidv4(),
+  example: '00000000-0000-4000-0000-000000000000',
```

Comment:
Is there a typo the "4" in the middle?

```suggestion
  example: '00000000-0000-0000-0000-000000000000',
```

Reply:
No, most of the tests had a fixed string like this, only few tests used the `uuidv4()`. I made everything the same string because snapshots were failing because of this